d2target/d2target.go

```diff
@@ -277,6 +277,8 @@ const (
 	TriangleArrowhead      Arrowhead = "triangle"
 	DiamondArrowhead       Arrowhead = "diamond"
 	FilledDiamondArrowhead Arrowhead = "filled-diamond"
+	CircleArrowhead        Arrowhead = "circle"
+	FilledCircleArrowhead  Arrowhead = "circle-filled"
```

Comment:
Do we need this distinction?
```
# transparent fill, just stroke
source-arrowhead: {
    style.stroke: red
}

# filled
target-arrowhead: {
    style.fill: blue
}
```

Reply:
Also, `diamond` is `filled-diamond` and here is `circle-filled`

Reply:
Fixed, was an oversight, great eyes and review ejulio.